File: nl.naturalis.nda.domain/src/main/java/nl/naturalis/nda/domain/GeoPoint.java
package nl.naturalis.nda.domain;

public class GeoPoint extends GeoShape {

	private double[] coordinates;


	public GeoPoint()
	{
		this.type = Type.POINT;
	}


	public GeoPoint(double longitude, double latitude)
	{
		this.type = Type.POINT;
		this.coordinates = new double[] { longitude, latitude };
	}


	public double[] getCoordinates()
	{
		return coordinates;
	}


	public void setCoordinates(double[] coordinates)
	{
		this.coordinates = coordinates;
	}

}

File: nl.naturalis.nba.client.php/model/GeoPoint.php
<?php

namespace nl\naturalis\nba\client\php\model;

class GeoPoint extends GeoShape {

	private $coordinates;

	// TODO
	// public GeoPoint()
	// {
	// this.type = Type.POINT;
	// }

	// public GeoPoint(double longitude, double latitude)
	// {
	// this.type = Type.POINT;
	// this.coordinates = new double[] { longitude, latitude };
	// }

	public function getCoordinates()
	{
		return $this->coordinates;
	}

	public function setCoordinates(array $coordinates)
	{
		$this->coordinates = $coordinates;
	}
}
